e2e-tests/README.md

```diff
@@ -11,10 +11,6 @@ These are end to end tests triggered via a CI job. You can run these tests local
 ## Running the Tests
 
 - `cd` to the test (e.g. `cd e2e-tests/gatsby-image`)
-- Install dependencies: `yarn` or `npm install`
-- OPTIONAL: Use [gatsby-dev-cli][gatsby-dev-cli] to link current changes in packages
-- Run the `test` script, e.g. `yarn test` or `npm test`
+- Run the `e2e-test` script: `../../script/e2e-test.sh`
```

Comment:
This will not work on Windows (at least not on vanilla one) :( but it should be preferred way for linux and mac (so we might need branch here)

Reply:
God damn Windows

Reply:
Additionally - `themes` e2e tests are additionally nested: `/e2e-tests/themes/development-runtime` - current script have hard coded `../..` so it wouldn't even work there (on top of the fact that path to `e2e-script` will need additional `../`) :(

Can we figure out so `e2e-test.sh` doesn't have `../..`, but instead have "one directory about `e2e-test.sh` file"?